client/lib/feed-post-store/post-batch-fetcher.js

```diff
@@ -88,8 +88,8 @@ assign( PostBatchFetcher.prototype, {
 					return;
 				}
 
-				containerId = +match[ 1 ];
-				postId = +match[ 2 ];
+				containerId = match[ 1 ];
```

Comment:
I know that sites can be fetched by their slug, but if /where we're doing that, it's more a bug than a feature. We should be storing things by blog ID, never by slug.